Content.Server/Radio/EntitySystems/GetInsteadIdCardNameSystem.cs

```diff
@@ -0,0 +1,62 @@
+using Content.Server.Radio.EntitySystems;
+using Content.Shared.Access.Components;
+using Content.Shared.Inventory;
+using Content.Shared.PDA;
+using Content.Shared.Silicons.Borgs.Components;
+using System.Globalization;
+
+public sealed class GetInsteadIdCardName : EntitySystem
+{
+    [Dependency] private readonly InventorySystem _inventorySystem = default!;
+
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<BorgChassisComponent, GetInsteadIdCardNameEvent>(OnGetBorgName);
+        SubscribeLocalEvent<InventoryComponent, GetInsteadIdCardNameEvent>(OnGetIdCardName);
+    }
+
+    private void OnGetBorgName(EntityUid uid, BorgChassisComponent component, ref GetInsteadIdCardNameEvent args)
+    {
+        if (HasComp<BorgChassisComponent>(uid) && TryPrototype(uid, out var borgPototype))
+        {
+            string borJob = borgPototype.Name.ToString();
+            args.Name = $"\\[{borJob}\\] ";
+        }
+        else
+            args.Name = string.Empty;
+    }
+
+    private void OnGetIdCardName(EntityUid uid, InventoryComponent invent, ref GetInsteadIdCardNameEvent args)
```

Comment:
И перенести в `ServerInventorySystem`
```suggestion
    private void OnGetIdCardName(Entity<InventoryComponent> ent, ref GetInsteadIdCardNameEvent args)
```